pandas/core/generic.py

```diff
@@ -10234,28 +10236,37 @@ def _doc_parms(cls):
 
 Returns
 -------
-%(outname)s : %(name1)s or %(name2)s (if level specified)
+%(outname)s : %(name1)s or %(name2)s
```

Comment:
```suggestion
%(name1)s or %(name2)s
```

We don't give a name to the output anymore. You can get rid of `outname` in the parameters passed I think